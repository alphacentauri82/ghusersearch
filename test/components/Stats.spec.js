import React from 'react';
import renderer from 'react-test-renderer';
import {
  StyleSheetTestUtils,
} from 'aphrodite';
import Stats from 'components/Stats';

describe('Component: Stats', () => {

  beforeAll(() => {
    StyleSheetTestUtils.suppressStyleInjection();
  });

  afterAll(() => {
    StyleSheetTestUtils.clearBufferAndResumeStyleInjection();
  });

  it('should render', () => {
    const stats = {
      Foo: 'bar',
      Bla: 'yo',
    };
    const component = renderer.create(
      <Stats
        stats={stats}
      />
    );
    expect(component).toMatchSnapshot();
  });

});
