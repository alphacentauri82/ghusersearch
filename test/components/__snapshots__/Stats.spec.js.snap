// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Stats should render 1`] = `
<ul
  className="u-flex"
>
  <li
    className="Stats_item_1p0a99 u-flex u-flexCol u-flexAlignItemsCenter"
  >
    <p
      className="Stats_value_176gzik"
    >
      bar
    </p>
    <p
      className="Stats_label_f9e8qy"
    >
      Foo
    </p>
  </li>
  <li
    className="Stats_item_1p0a99 u-flex u-flexCol u-flexAlignItemsCenter"
  >
    <p
      className="Stats_value_176gzik"
    >
      yo
    </p>
    <p
      className="Stats_label_f9e8qy"
    >
      Bla
    </p>
  </li>
</ul>
`;
